src/wp-content/themes/twentytwentytwo/block-template-parts/header.html

```diff
@@ -0,0 +1,12 @@
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"bottom":"var(--wp--custom--spacing--large, 8rem)","top":"var(--wp--custom--spacing--small, 1.25rem)"}}},"layout":{"type":"flex","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--custom--spacing--small, 1.25rem);padding-bottom:var(--wp--custom--spacing--large, 8rem)"><!-- wp:group {"layout":{"type":"flex"}} -->
+<div class="wp-block-group">
+<!-- wp:site-logo {"width":64} /-->
+
+<!-- wp:site-title {"style":{"typography":{"fontStyle":"italic","fontWeight":"400"}}} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:navigation {"itemsJustification":"right","isResponsive":true} -->
+<!-- wp:page-list {"isNavigationChild":true,"showSubmenuIcon":true,"openSubmenusOnClick":false} /-->
+<!-- /wp:navigation --></div>
+<!-- /wp:group -->
```

Comment:
Each file should have a new, blank line at the end to prevent problems with some systems that expect every line to end with a newline `\n`. Many of the `.html` template files need this fixed.